lib/src/excel.dart

```diff
@@ -216,17 +218,24 @@ class ExcelDart {
       // This formula calculates:
       // (spent / budget) / (current duration / flight duration)
       //
-      // Flight duration is calculated by (end date - start date + 1), and
-      // current duration is calculated by (now - start date + 1), so it is
-      // guaranteed that divide by zero will not happen.
-      /// TODO: calculate difference at hours level if it is shorter than a day.
-      /// Issue: https://github.com/googleinterns/dv360-excel-plugin/issues/57
+      // If current duration or flight duration is less than 1 day, then
+      // the formula calculates:
+      // (spent / budget) / (time elapsed since flight start / 24)
+      // Time elapsed has the granularity of hours and minutes.
+      // i.e. TEXT("1/1/2020 10:20:00 AM" - "1/1/2020 12:00:00 AM", "h.mm")
+      //      = 10.2
       final formula = ''' 
-       ($_spentColumn/$_budgetColumn) / 
-       (
-         (DATEDIF($_startDateColumn, NOW(), "D") + 1) /
-         (DATEDIF($_startDateColumn, $_endDateColumn, "D") + 1)
-       ) < $_thresholdAddress
+      IF(OR(DATEDIF($_startDateColumn, $_endDateColumn, "D") = 0, 
+            DATEDIF($_startDateColumn, NOW(), "D") = 0),
+         ($_spentColumn/$_budgetColumn) / 
+         (
+           TEXT(NOW()-$_startDateColumn, "h.mm") / 24
+         ), 
+         ($_spentColumn/$_budgetColumn) /
+         (
+           DATEDIF($_startDateColumn, NOW(), "D") /
```

Comment:
Is this only for active IOs? If not (now - startDate) can be -ve for segments in future.

Reply:
Yes, we are only querying for IOs that have entity status of active. And for all the active IOs, i will also filter out those that are not in flight.